packages/cli-plugin-metro/launchPackager.sh

```diff
@@ -0,0 +1,4 @@
+THIS_DIR=$(cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")")" && pwd)
+source "$THIS_DIR/.packager.env"
+cd $PROJECT_ROOT
+$REACT_NATIVE_PATH/cli.js start
```

Comment:
Should we carry over this block from `launchPackager.command`?
```sh
if [[ -z "$CI" ]]; then
  echo "Process terminated. Press <enter> to close the window"
  read -r
fi
```

Reply:
Yes, we should.